From f0b7333c1b0c5609c4a0358a4fbe22c9ba77d080 Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Thu, 14 Aug 2014 13:52:34 +0100
Subject: [PATCH 296/351] mfd: florida: Remove default for volatile register

There is no point in having a default for Mic Detect 4 as it is a
volatile register and marked as such.

Change-Id: I01efdbd8bab3d165e523b29059ce8237f15b7e97
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
---
 drivers/mfd/florida-tables.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/mfd/florida-tables.c b/drivers/mfd/florida-tables.c
index 23bca48..abd5517 100644
--- a/drivers/mfd/florida-tables.c
+++ b/drivers/mfd/florida-tables.c
@@ -766,7 +766,6 @@ static const struct reg_default florida_reg_default[] = {
 	{ 0x000002A7, 0x372C },    /* R679   - Mic Detect Level 2 */
 	{ 0x000002A8, 0x1422 },    /* R680   - Mic Detect Level 3 */
 	{ 0x000002A9, 0x300A },    /* R681   - Mic Detect Level 4 */
-	{ 0x000002AB, 0x0000 },    /* R683   - Mic Detect 4 */
 	{ 0x000002C3, 0x0000 },    /* R707   - Mic noise mix control 1 */
 	{ 0x000002CB, 0x0000 },    /* R715   - Isolation control */
 	{ 0x000002D3, 0x0000 },    /* R723   - Jack detect analogue */
-- 
1.7.9.5

